sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1259,13 +1262,14 @@ SUBROUTINE MDL2P(iostatusD3D)
               ENDDO
             ENDDO
 !
-            IF(MODELNAME == 'GFS' .or. MODELNAME == 'FV3R')THEN
-              CALL CALRH_GFS(EGRID2(1,jsta),TSL(1,jsta),QSL(1,jsta),EGRID1(1,jsta))
-            ELSEIF (MODELNAME == 'RAPR')THEN 
-              CALL CALRH_GSD(EGRID2(1,jsta),TSL(1,jsta),QSL(1,jsta),EGRID1(1,jsta))
-            ELSE
-              CALL CALRH(EGRID2(1,jsta),TSL(1,jsta),QSL(1,jsta),EGRID1(1,jsta))
-            END IF 
+            CALL CALRH(EGRID2(:,jsta:jend),TSL(:,jsta:jend),QSL(:,jsta:jend),EGRID1(:,jsta:jend))
```

Comment:
^^^

Reply:
@JesseMeng-NOAA Could you keep the original style like
CALL CALRH(EGRID2(1,jsta),TSL(1,jsta),QSL(1,jsta),EGRID1(1,jsta))